RELEASE.md

```diff
@@ -109,3 +109,49 @@ We use [github's Releases UI](https://github.com/solana-labs/solana/releases) fo
    build` to cause a refresh of `Cargo.lock`.
 1. Push your Cargo.toml change and the autogenerated Cargo.lock changes to the
    release branch.
+
+### Update software version running on testnet.solana.com
+
+The testnet running on testnet.solana.com is set to use a fixed release tag which is set in the Buildkite testnet-management pipeline.  This tag needs to be updated and the testnet restarted after a new release tag is created.
+
+Testnet runs on the stable channel, one branch behind the newly created branch.
+
+##### Update testnet schedules
+
+Go to https://buildkite.com/solana-labs and click through: Pipelines -> testnet-management -> Pipeline Settings -> Schedules
+Or just click here:  https://buildkite.com/solana-labs/testnet-management/settings/schedules
+
+There are two scheduled jobs for testnet: a daily restart and an hourly sanity-or-restart. \
+https://buildkite.com/solana-labs/testnet-management/settings/schedules/0efd7856-7143-4713-8817-47e6bdb05387
+https://buildkite.com/solana-labs/testnet-management/settings/schedules/2a926646-d972-42b5-aeb9-bb6759592a53
+
+On each schedule:
```

Comment:
Let's remove "stable channel" mention here too, we mostly use beta tags a lot for "testnet" right now.

eg:
```
On each schedule:
1.  Set TESTNET_TAG environment variable to the desired release tag
2.  Set the Build Branch to the new branch that TESTNET_TAG is from.
```